data/modules/SearchRescue/SearchRescue.lua

```diff
@@ -1967,6 +1977,18 @@ local onUpdateBB = function (station)
 		end
 	end
 
+	-- force ad creation for debugging
+	-- local num = 3
+	-- for _ = 1,num do
+	--	makeAdvert(station, 1, closestplanets)
+	--	makeAdvert(station, 2, closestplanets)
+	-- 	makeAdvert(station, 3, closestplanets)
+	-- 	makeAdvert(station, 4, closestplanets)
+	-- 	makeAdvert(station, 5, closestplanets)
+	-- 	makeAdvert(station, 6, closestplanets)
+	-- 	makeAdvert(station, 7, closestplanets)
+	-- end
+
 	-- trigger new ad creation if appropriate
```

Comment:
I'm not sure we normally save debug print out in the finished file, but I guess it doesn't hurt. I have no real opinion one way or the other

Reply:
I've always left this in for faster debugging. But I can remove it if you prefer.